src/constants.ts

```diff
@@ -1,3 +1,4 @@
+// Credit to https://github.com/OpenZeppelin/openzeppelin-subgraphs
```

Comment:
Should call out the license too, MIT in this case

Reply:
Will add

Reply:
// Credit to <URL>, included under <LICENSE>

Reply:
done in https://github.com/valorem-labs-inc/valorem-subgraph/commit/ecb0958c55a60effddb94cc5d773b72580758013